### PR TITLE
Enable swagger to sdk pipeline for Ruby in master branch

### PR DESCRIPTION
**Pull request by vishrutshah**
Enable swagger to sdk pipeline for Ruby in master branch

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vishrutshah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: .travis.yml [hunk 1]
```diff
@@ -17,4 +18,5 @@ script:
   - DOCKER_CMD="docker run --rm --env-file /tmp/env_file -e GH_TOKEN -v $PWD:/git-restapi/ lmazuel/swagger-to-sdk"
   - if [[ $MODE == 'python' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-for-python --pr-repo-id Azure/azure-sdk-for-python -o dev -v; fi
   - if [[ $MODE == 'node' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-for-node --pr-repo-id Azure/azure-sdk-for-node -o master -v; fi
+  - if [[ $MODE == 'ruby' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-ruby --pr-repo-id Azure/azure-sdk-ruby -o swagger-to-sdk -v; fi
```
Comment (lmazuel) [inline .travis.yml]: Typo (missing the for)
- `AutorestCI/azure-sdk-for-ruby`
- `Azure/azure-sdk-for-ruby`

Comment (lmazuel) [inline .travis.yml]: `-o swagger-to-sdk` will push in this branch the generated work.
- Is this the expected behaviour?
- Does this branch exists?

Comment (vishrutshah) [inline .travis.yml]: Actually we named the repo as https://github.com/Azure/azure-sdk-ruby :)

Comment (vishrutshah) [inline .travis.yml]: Yup I created that branch(swagger-to-sdk) to try this pipeline for ruby for now. Once i think, things are good we'd switch it to master branch.

**Comment by lmazuel** [comment]
@vishrutshah The build is green, but we didn't actually test the generation it self. The script is looking for the list of swagger files changed by this PR. Since there is swagger files changed by this PR, no build are tested:
```
INFO:__main__:Skip file not in PR authorization
```

I suggest you to launch the script manually from your computer, using `vishrutshah:swagger_to_sdk_ruby` as your local branch. You should be able to test the real behaviour.

**Comment by vishrutshah** [comment]
@lmazuel I've pushed [swagger_to_sdk_config.json](https://github.com/Azure/azure-sdk-ruby/commit/f5286f0bb7992f8f6fe127f0b483910dc3b01780) into master branch to enable this pipeline. How do you sync https://github.com/AutorestCI/azure-sdk-ruby from https://github.com/Azure/azure-sdk-ruby.

**Pull request merged**